tricolour/tests/test_acceptance.py

```diff
@@ -14,43 +14,16 @@
 
 from pyrap.tables import table as tbl
 import numpy as np
-import requests
+import gdown
```

Comment:
nice, I didn't know about this.

Reply:
Yup damned google innovation has broken basic requests compatibility in their API...